_data/projects/alda.yml

```diff
@@ -0,0 +1,12 @@
+name: Alda
+desc: A general-purpose, beginner-friendly music programming language
+site: https://github.com/alda-lang/alda
+tags:
+- music
+- language
+- clojure
+- java
+- zeromq
+upforgrabs:
+  name: low-hanging fruit
```

Comment:
Looking at the issue list, it seems like both this label or `up-for-grabs` are used. Feel free to use either one.